docs/sdk-json-proposal/SDK_API_WORKFLOW.md

```diff
@@ -62,6 +62,14 @@
   }
 }
 ```
+| BCS(json string) | MTL transport_format(enum st20_fmt) | MTL {in/out}put_format(enum st_frame_fmt) | FFmpeg (enum PixelFormat) |FFmpeg (string name) | RFC 4715 compliant | Note |
```

Comment:
[RFC 4715](https://datatracker.ietf.org/doc/rfc4715/) has a title *The Integrated Services Digital Network (ISDN) Subaddress Encoding Type for tel URI*. Is it the right document?

Reply:
Oh, sorry my bad, it should be 4175 _RTP Payload Format for Uncompressed Video_

Reply:
Thank you :)